build/patches/Re-introduce-kWebAuthCable.patch

```diff
@@ -76,30 +75,14 @@ diff --git a/chrome/browser/webauthn/chrome_webauthn_browsertest.cc b/chrome/bro
      // This makes it a little easier to compare against.
      trace_ << std::endl;
    }
-@@ -697,6 +698,7 @@ class WebAuthnCableSecondFactor : public WebAuthnBrowserTest {
+@@ -695,6 +696,7 @@ class WebAuthnCableSecondFactor : public WebAuthnBrowserTest {
    };
  
   protected:
 +  base::test::ScopedFeatureList scoped_feature_list_;
    std::ostringstream trace_;
    AuthenticatorRequestDialogModel* model_ = nullptr;
  };
```

Comment:
this one is missing:
```
diff --git a/content/browser/webauth/authenticator_common.cc b/content/browser/webauth/authenticator_common.cc
--- a/content/browser/webauth/authenticator_common.cc
+++ b/content/browser/webauth/authenticator_common.cc
@@ -298,8 +298,10 @@ base::flat_set<device::FidoTransportProtocol> GetWebAuthnTransports(
     transports.insert(device::FidoTransportProtocol::kInternal);
   }

-  transports.insert(
-      device::FidoTransportProtocol::kCloudAssistedBluetoothLowEnergy);
+  if (base::FeatureList::IsEnabled(features::kWebAuthCable)) {
+    transports.insert(
+        device::FidoTransportProtocol::kCloudAssistedBluetoothLowEnergy);
+  }

   // kAndroidAccessory doesn't work on Windows because of USB stack issues.
   // Note: even if this value were inserted it wouldn't take effect on Windows
```
Or did you intentionally take it out?

Reply:
It does not exist: https://github.com/chromium/chromium/blob/106.0.5249.72/content/browser/webauth/authenticator_common.cc#L293

Reply:
OK, I don't know that patch nor would I know how to test it.